app/views/users/user.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1> Here is an author with its corresponding bio and his/her posts
+</h1>
```

Comment:
- In making the change, this file needs to be something like this: `app/views/users/show.html.erb` instead of `app/views/users/users.html.erb` because it must correspond to the controller actions